tex/gsp-default.tex

```diff
@@ -210,8 +210,12 @@
 \grecreatedim{initialraise}{0 cm}{scalable}%
 % Space between lines in the annotation
 \grecreatedim{annotationseparation}{0.05cm}{scalable}%
-% Amount to raise (positive) or lower (negative) the annotations from the default position (base line of top annotation aligned with top line of staff)
+% Amount to raise (positive) or lower (negative) the annotations from the default position
 \grecreatedim{annotationraise}{0cm}{scalable}%
+% Space between lines in the commentary
+\grecreatedim{commentaryseparation}{0.05cm}{scalable}%
+% Amount to raise (positive) or lower (negative) the commentary from the default position (base line of bottom commentary aligned with top line of staff)
+\grecreatedim{commentaryraise}{0.4cm}{scalable}%
```

Comment:
1. I think it is more desirable yes

2. I don't really understand your point... what unexpected? Also, easiness seems important for you, and I'm affraid changing a dimension in a group actually involves very difficult notions for a beginner... And when the commentary is close to the score, it's necessary to raise it by hand if there are some high notes.

3. Something else I forgot: are you sure your implementation never allows page breaks between commentary and score? (that was a problem with the old implementation)